org.eclipse.xtext.xtext.ui.examples/projects/fowlerdsl/org.eclipse.xtext.example.fowlerdsl.ui/src/org/eclipse/xtext/example/fowlerdsl/ui/labeling/StatemachineDescriptionLabelProvider.java

```diff
@@ -1,30 +1,28 @@
 /*******************************************************************************
- * Copyright (c) 2013, 2019 itemis AG (http://www.itemis.eu) and others.
+ * Copyright (c) 2019 itemis AG (http://www.itemis.eu) and others.
  * All rights reserved. This program and the accompanying materials
  * are made available under the terms of the Eclipse Public License v1.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-v10.html
  *******************************************************************************/
-package org.eclipse.xtext.example.fowlerdsl.ui.labeling
+package org.eclipse.xtext.example.fowlerdsl.ui.labeling;
 
-import org.eclipse.xtext.ui.label.DefaultDescriptionLabelProvider
-
-//import org.eclipse.xtext.resource.IEObjectDescription
+import org.eclipse.xtext.ui.label.DefaultDescriptionLabelProvider;
 
 /**
  * Provides labels for IEObjectDescriptions and IResourceDescriptions.
- *
- * See https://www.eclipse.org/Xtext/documentation/310_eclipse_support.html#label-provider
+ * 
+ * https://www.eclipse.org/Xtext/documentation/310_eclipse_support.html#label-provider
  */
-class StatemachineDescriptionLabelProvider extends DefaultDescriptionLabelProvider {
+public class StatemachineDescriptionLabelProvider extends DefaultDescriptionLabelProvider {
 
 	// Labels and icons can be computed like this:
-
-//	override text(IEObjectDescription ele) {
-//		ele.name.toString
+	
+//	String text(IEObjectDescription ele) {
+//		return ele.getName().toString();
 //	}
-//
-//	override image(IEObjectDescription ele) {
-//		ele.EClass.name + '.gif'
+//	 
+//	String image(IEObjectDescription ele) {
```

Comment:
Access modifier? `@Override`?

Reply:
this is the default template. please create an issue against the generator